common/src/main/java/tachyon/exception/ExceptionMessage.java

```diff
@@ -116,6 +116,9 @@
   LINEAGE_INPUT_FILE_NOT_EXIST("The lineage input file {0} does not exist"),
   LINEAGE_OUTPUT_FILE_NOT_EXIST("No lineage has output file {0}"),
 
+  // client
+  INCOMPATIBLE_VERSION("{0}Client version {1} is not compatible with server version {2}"),
```

Comment:
nit: I'd prefer 'Client' to be passed in so that the final word isn't split between two files